charts/newrelic-k8s-metrics-adapter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
-version: 1.1.2
-appVersion: 0.3.0
```

Comment:
Can you revert this app & chart version bump and we'll just pull this into the next release?